src/lib/duration/valid.js

```diff
@@ -0,0 +1,24 @@
+var ordering = ['year', 'quarter', 'month', 'week', 'day', 'hour', 'minute', 'second', 'millisecond'];
+
+export default function isDurationValid(m) {
+    for (var key in m) {
+        if (ordering.indexOf(key) === -1 ||
+        m[key] !== undefined && isNaN(parseInt(m[key]))) {
```

Comment:
I'm amazed jscs doesn't catch this but this line needs to be indented 8 more spaces (continuation indent).